tests/ut/ir/transforms/test_expand_mixed_kernel_a5.py

```
@@ -3585,6 +3585,153 @@ def main_incore_0(
+                        z_1 = pl.matmul(x_left_1, w_right_1)
+                        z_vec_1 = pl.move(
+                            z_1,
+                            target_memory=pl.MemorySpace.Vec,
+                            blayout=pl.TileLayout.row_major,
+                            slayout=pl.TileLayout.none_box,
+                        )
+                        q_then = pl.store(z_vec_1, [0, 0], q_iter)
+                        q_phi = pl.yield_(q_then)
+                    else:
+                        q_phi = pl.yield_(q_iter)
+                    p_rv, q_rv = pl.yield_(p_phi, q_phi)  # noqa: F841
```

Ruff (0.16.1)

[warning] Unpacked variable `q_rv` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)